Formula/ldc.rb

```diff
@@ -26,35 +26,30 @@ class Ldc < Formula
   depends_on "cmake" => :build
   depends_on "libconfig" => :build
   depends_on "pkg-config" => :build
-  depends_on "llvm@14"
+  depends_on "llvm@14" # LLVM 15 issue: https://github.com/ldc-developers/ldc/issues/4042
 
   uses_from_macos "libxml2" => :build
 
-  on_linux do
-    # Superenv does not support building with a versioned LLVM.
-    depends_on "llvm" => [:build, :test]
-  end
-
-  fails_with :gcc
```

Comment:
Not sure if there are any issues building this with GCC.

Having 2 LLVM versions as dependencies is a bit brutal if you try to build everything from source.

Alternative would be to manually set `CMAKE_{C,CXX}_COMPILER` to path of clang/clang++